pandas/util/testing.py

```diff
@@ -931,13 +947,13 @@ def raise_assert_detail(obj, message, left, right, diff=None):
 
 
 def assert_numpy_array_equal(
-    left,
-    right,
-    strict_nan=False,
-    check_dtype=True,
-    err_msg=None,
-    check_same=None,
-    obj="numpy array",
+    left: np.ndarray,
+    right: np.ndarray,
+    strict_nan: bool = False,
+    check_dtype: bool = True,
+    err_msg: Optional[str] = None,
+    check_same: Optional[str] = None,
+    obj: str = "numpy array",
 ):
```

Comment:
Could annotate return of None here as well

Reply:
can you update this one

Reply:
done